### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by Vishvakb**

### CODE REVIEW

File: Untitled-1.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
+
+    <title>Camber Racing</title>
+  </head>
+  <body>
+    <h1>  </h1>
```
Comment (Ritvick-me) [inline Untitled-1.html]: Why is an empty H1 tag there? Make sure there are no such empty tags in the code.

File: Untitled-1.html [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    <div class="container"></div>
```
Comment (Ritvick-me) [inline Untitled-1.html]: Why is there an empty div.container?

File: Untitled-1.html [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    <div id="carouselExampleIndicators" class="carousel slide" data-ride="carousel">
+        <ol class="carousel-indicators">
+          <li data-target="#carouselExampleIndicators" data-slide-to="0" class="active"></li>
```
Comment (Ritvick-me) [inline Untitled-1.html]: Change data-target value to something meaningful.

File: Untitled-1.html [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+            <img src="CR16.JPEG" class="d-block w-100">
+          </div>
+          <div class="carousel-item">
+            <img src="CR14.JPG" class="d-block w-100">
+          </div>
+          <div class="carousel-item">
+            <img src="CR15.JPG" class="d-block w-100">
+          </div>
+        </div>
+        <a class="carousel-control-prev" href="#carouselExampleIndicators" role="button" data-slide="prev">
```
Comment (Ritvick-me) [inline Untitled-1.html]: Change #carouselExampleIndicators in href to something meaningful.

**Review by Ritvick-me — COMMENTED** [review]
The PR should have a more "Informative heading" than "Add files via upload". Please take care of it in the next PRs.

File: Untitled-1.html (hunk 1 above)
Comment (Vishvakb) [reply Untitled-1.html]: Done

**Pull request merged**